controllers/consoleplugin/consoleplugin_objects.go

```diff
@@ -208,18 +208,6 @@ func (b *builder) podTemplate(cmDigest string) *corev1.PodTemplateSpec {
 		})
 	}
 
-	// ensure volumes are up to date
```

Comment:
I think this code is not necessary but I don't want to introduce a regression so close to the release, so I'm only removing it from `main`, [not in the backport PR](https://github.com/netobserv/network-observability-operator/pull/670/files), to play safe.
Note that we discussed this already a while ago: https://github.com/netobserv/network-observability-operator/pull/477/files?diff=split&w=1#r1384571479 - I don't see why this code is duplicated, and if there's a reason I can bring it back but it should be better documented and ideally covered by a test that show-case it.